### DISCUSSION TITLE
downloadhelper not working

### DISCUSSION THREAD

**Discussion by noddy33** [aclap-dev/video-downloadhelper#900] [Help and Q&A]
### Your question

I have windows 7 and since downloading the latest version for firefox when for example I try to download a YouTube video zero % downloads an a cursor just spins round and round. When I try to use an older version now the links are queued and nothing downloads. I used to have no problem on any downloads in the past. Thanks.

### Addon & CoApp version and details

9.0.2.3

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Normally, I would scold you for having completely ignored the problem reporting process.  What you have posted is utterly useless for diagnosing anything.  You have not provided enough detail for anybody else to try to replicate your problem.  That is the heart of problem diagnosis & resolution.  You have to provide enough information so somebody else can duplicate your results.  That is the primary underpinning of science: the results have to be repeatable by anybody.  But you haven't told us anything.

Normally I would just stop there.  But it so happens that we got a brand new release of VDH today.  I want to try it out on something.  So I will continue.

Here is an important group of facts.  You failed to provide these.  That is part of why your problem report is useless.  I am running Windows 11 Pro 64-bit version 23H2 build 22631.3737, Firefox 127.0 64-bit, licensed VDH 9.0.2.12a1 beta, CoApp 2.0.19.

I used this for demonstration purposes:

https://www.youtube.com/watch?v=hyMfJa8qN2Q

That's the other part of your problem report that's missing.  I don't care if you think every video on YouTube is giving you problems.  Nobody can test every video on YouTube.  You need to tell people at least one & perhaps a handful of URLs on YouTube that are giving you trouble.  Like I just did.  Except it's not giving me trouble.  Here's the proof.

When I first visited that page, VDH recognized a video.  I decided to get the highest possible resolution on offer for this video.  So I dropped down the list of variants & saw there was a 4K variant on offer:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/dcfc8873-f251-4e83-8657-9171d6b84261)

So I downloaded it.

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/1669b451-9c5f-4285-b036-15e453284eee)

Full disclosure.  I actually forgot to capture that image first time through.  So I went back to regenerate that state of affairs & capture that screenshot.  You'll have to trust me that on my first pass through these steps, that's exactly what things looked like.  You can tell that image is out of sequence because almost all of these images show a date/time stamp.  Look in the bottom right corner.

So once I launched my Download As function, it progressed quite nicely.

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/2848ca55-169f-4488-91c4-b7bfac68250e)

That's a pretty good download speed.  That's not just really good for YouTube, that would be a great speed on any site.  If you can't read that -- I can't -- just click the image.  GitHub will open the image in its own browser window where you can use the browser's magnifier (Ctrl-+) to make it big enough to read.  That is true of any image in any discussion in any forum on GitHub.

Eventually, the download finished.

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/33abaff7-dcb5-466d-abc1-257abc602cd9)

This is what I had as a result.

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/61396093-d4f9-42fe-b5f8-343c5e468140)

And here it is playing in VLC.

![#06](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/7940e3c1-bdd6-4651-ae65-80de1f1f199e)

You can see the video is just fine.  You'll have to trust me the audio was equally just fine.  I didn't so much watch this as just listen to it.  It's a particular favorite piece of music for me so I did listen all the way through but only occasionally looked at it.  It's a perfectly excellent performance.  I've never heard of this conductor or soloist before.  Warsaw Philharmonic . . . yeah, I've heard of them.  But I wouldn't hesitate to pay attention to these folks in future.

The bottom line here is that our brand new VDH works as well as it ever has on YouTube.  Reports to the contrary need to be backed up with examples.  Let's see if you learned how to provide a proper problem description.  If you have, there's a chance we can get to the bottom of your issues.

**Comment by noddy33** [comment]
Ok no need to scold me I'm not very tech savvy and wasn't sure of the
reporting process. But thanks for responding and I will just have to get
used to not using videodownloader. Thanks.
